### PR TITLE
Feat/1206361027512394 upgrade polkadot v1.7.2 async

### PR DESCRIPTION
**Pull request by sfffaaa**
1. Upgrade to 1.1.0 first
2. Upgrade to 1.7.2
3. Enable the async backing
4. Haven't changed the implementation by the impact analysis; will create another task to do that

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206361027512394

### CODE REVIEW

File: runtime/common/src/xcm_impls.rs [hunk 1]
```diff
@@ -112,10 +114,19 @@ impl<T: ExecutionPaymentRate, R: TakeRevenue> WeightTrader for FixedRateOfForeig
...
-		if let Some((asset_location, _)) = self.asset_location_and_units_per_second {
+		if let Some((asset_location, _)) = self.asset_location_and_units_per_second.clone() {
 			if self.consumed > 0 {
 				R::take_revenue((asset_location, self.consumed).into());
 			}
 		}
 	}
 }
+
+// [TODO] Comment it out because we won't use that
```
Comment (DocteurPing) [inline runtime/common/src/xcm_impls.rs]: Can't we just remove it if we won't use it ?

File: node/src/parachain/service.rs [hunk 2]
```diff
@@ -78,11 +86,23 @@ pub fn frontier_database_dir(config: &Configuration, path: &str) -> std::path::P
...
+) -> Result<fc_db::Backend<Block>, String>
+where
+	C: ProvideRuntimeApi<Block> + StorageProvider<Block, BE> + AuxStore,
+	C: HeaderBackend<Block> + HeaderMetadata<Block, Error = BlockChainError>,
+	C: Send + Sync + 'static,
+	C::Api: fp_rpc::EthereumRuntimeRPCApi<Block>,
+	BE: Backend<Block> + 'static,
+	BE::State: StateBackend<BlakeTwo256>,
+{
+	// let frontier_backend = match rpc_config.frontier_backend_config {
```
Comment (DocteurPing) [inline node/src/parachain/service.rs]: Should we delete this comment ?

File: node/src/parachain/service.rs [hunk 3]
```diff
@@ -101,7 +121,21 @@ pub fn open_frontier_backend<C: sp_blockchain::HeaderBackend<Block>>(
 					return Err("Supported db sources: `rocksdb` | `paritydb` | `auto`".to_string()),
 			},
 		},
-	)?)))
+	)?);
+	// }
```
Comment (DocteurPing) [inline node/src/parachain/service.rs]: Should we delete this ?

Comment (sfffaaa) [reply node/src/parachain/service.rs]: Deleted

File: node/src/parachain/service.rs (hunk 2 above)
Comment (sfffaaa) [reply node/src/parachain/service.rs]: Deleted

File: runtime/common/src/xcm_impls.rs (hunk 1 above)
Comment (sfffaaa) [reply runtime/common/src/xcm_impls.rs]: Deleted

**Review by DocteurPing — APPROVED** [review]

**Pull request merged**